app/assets/javascripts/mobile/publisher.js

```diff
@@ -48,4 +48,76 @@ $(document).ready(function(){
   });
 
   new Diaspora.MarkdownEditor("#status_message_text");
+
+  $(".dropdown-menu > li").bind("tap click", function(evt) {
```

Comment:
The code you add here is mostly copy-pasted code from [`app/assets/javascripts/app/views/publisher/aspect_selector_view.js`](https://github.com/diaspora/diaspora/blob/b263cf7e6db3653e6751a47526c941d5c4518910/app/assets/javascripts/app/views/publisher/aspect_selector_view.js#L21). Such approach in programming is highly discouraged. Instead you should create a helper class which could be shared between mobile and desktop versions, like we have it for the [markdown editor](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/helpers/markdown_editor.js).

Reply:
@sebastienadam can you please have a look at that remark? Then we will do the review ;)

Reply:
I could try to fix this myself too, but I better trust the judgement of Denis here